templates/squid.conf.header.erb

```diff
@@ -59,7 +59,11 @@ via                           <%= @via?'on':'off' %>
 httpd_suppress_version_string <%= @httpd_suppress_version_string?'on':'off' %>
 <% end -%>
 <% unless @forwarded_for.nil? -%>
+<% if [true, false].include? @forwarded_for -%>
 forwarded_for                 <%= @forwarded_for?'on':'off' %>
+<% else -%> 
```

Comment:
You can drop the `<% unless @forwarded_for.nil? -%>` line if you make this an elsif:

```suggestion
<% elsif !@forwarded_for.nil? -%> 
```